crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/controller/StatelessMeansAssessmentController.java

```diff
@@ -44,8 +47,13 @@ public ResponseEntity<StatelessApiResponse> invoke(@Parameter(description = "Sta
     ) @Valid @RequestBody StatelessApiRequest meansAssessment) {
         var apiAssessment = meansAssessment.getAssessment();
         Map<AgeRange, Integer> childGroupings = getChildGroupings(apiAssessment.getDependantChildren());
+
+        final var criteriaEntry = assessmentCriteriaService.getAssessmentCriteria(
+                apiAssessment.getAssessmentDate(), apiAssessment.getHasPartner(), false
+        );
```

Comment:
@starswan This call to getAssessmentCriteria is moved from inside of SMAS/MAS and moved into the controller. What's the reason for this? (This PR aims to resolve "stateless service class is too tightly bound to the interface definitions of the stateless API", but I don't see the link with this change)

Reply:
The plan was for StatelessAssessmentService.execute() to do this call:
<img width="702" alt="Screenshot 2023-11-06 at 12 55 44" src="https://github.com/ministryofjustice/laa-crime-means-assessment/assets/307612/584774bb-6537-450e-8e8c-08345598a534">
I'm interested why we're departing from that.

Reply:
I think I know that this change is coming - stateless / cfe-crime knows to ignore contrary interest partners, so it seemed sensible to move this line out of the service rather than pass the 3 parameters required to obtain it. I've put it back for now

Reply:
Thanks for moving that back - we can look again at the contrary interest thing at another point 👍